config/initializers/content_security_policy.rb

```diff
@@ -98,6 +98,8 @@
     style_src: %w('self' blob: https://maxcdn.bootstrapcdn.com
                       https://*.appcues.com https://*.appcues.net https://fonts.googleapis.com https://fonts.google.com 'unsafe-inline'),
     upgrade_insecure_requests: true, # see https://www.w3.org/TR/upgrade-insecure-requests/
+    worker_src: %w('self' blob:),
+    child_src: %w('self' blob:)
```

Comment:
Couple things:
1) _Apparently_, these values actually aren't being honored in our CSP setup for some reason (not present on any requests)
2) I took out the values completely and replays were still being sent/sampled, so I guess they aren't needed
3) Regardless, I did a quick check on https://csp-evaluator.withgoogle.com/ and both `worker-src` and `child-src` with `'self' blob;` are green - no security implications

Anyway, I'm going to remove them from this PR since they aren't being used.

Reply:
@eweitz [b1a349d](https://github.com/broadinstitute/single_cell_portal_core/pull/1800/commits/b1a349d7a6fc3910dc152a9385ccf5487383a4f8)